src/components/data-processing/StatusIndicator.jsx

```diff
@@ -118,15 +117,15 @@ const StatusIndicator = (props) => {
         <Text strong>
           Your data processing is
           {' '}
-          {statusIndicators[status].title}
+          {statusIndicators[status]?.title || 'loading'}
```

Comment:
1. Why are you adding `loading` as default here? the backend status default is set in `/Users/ahriman/repos/github.com/biomage-ltd/ui/src/redux/selectors/backendStatus/getBackendStatus.js`. If the status is `null` it might be because it has not run right, instead of loading?

2. Did you have a look at the status indicator with loading? the other status have specific formatting (like coloring the status name, etc... Wouldn't it be better to add another `statusIndicator` case instead of doing `?` conditional chaining?

Reply:
The value of `status` is `null` because the backend status data has not been loaded. We do not use a `switch` statement to give the status, but use an object keyed with status to access the details. I just noticed that we can use `null` as the key. Should I do this?